From 10d9a7d4830f82579af014fe0f83ad113426b173 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 5 Nov 2020 11:20:09 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-evb-ind: remove
 regulator-xxx-microvolt for SWITCH pmic node

Switch is just a function of switching. There is no voltage setting
function. Voltage getting is the supply voltage.

Fixes: 3fc99e38fdbf ("regulator: resolve supply after creating regulator")
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: Ia45677a24fbfb831226563c56bcb26885bb17db1
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
index c6d841bd5c20..6f8ceb71f06d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
@@ -712,22 +712,15 @@
 			vcc5v0_usb: SWITCH_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <5000000>;
-				regulator-max-microvolt = <5000000>;
-
 				regulator-name = "vcc5v0_usb";
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
 			vccio_3v3: SWITCH_REG2 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
-
 				regulator-name = "vccio_3v3";
 				regulator-state-mem {
 					regulator-off-in-suspend;
-- 
2.35.3

